docs/index.rst

```diff
@@ -35,31 +30,22 @@ with different methods for different authentication scenarios.
 
 PublicClientApplication
 -----------------------
+
 .. autoclass:: msal.PublicClientApplication
    :members:
+   :inherited-members:
 
 ConfidentialClientApplication
 -----------------------------
-.. autoclass:: msal.ConfidentialClientApplication
-   :members:
-
 
-Shared Methods
---------------
-Both PublicClientApplication and ConfidentialClientApplication
-have following methods inherited from their base class.
-You typically do not need to initiate this base class, though.
-
-.. autoclass:: msal.ClientApplication
+.. autoclass:: msal.ConfidentialClientApplication
    :members:
-
-   .. automethod:: __init__
```

Comment:
Hi @tonybaloney , remember this PR we did last month? Today I noticed that [the new documentation for `PublicClientApplication`](https://msal-python.readthedocs.io/en/stable/#publicclientapplication) and [`ConfidentialClientApplication`](https://msal-python.readthedocs.io/en/stable/#confidentialclientapplication) lose [the detailed documentation on `__init__()` available from older docs](https://msal-python.readthedocs.io/en/testing-docs/#shared-methods). Would you mind if we bring back the `Shared Methods` approach here (while keeping your new theme)?